Sources/Nakama/2.XPackets/NakamaError.swift

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Heroic Labs
+ * Copyright 2018 Toggl
```

Comment:
Can you ensure that the Copyright Information is correct here?

Reply:
yep my bad

Reply:
@mofirouz we have more changes coming to this client (related to administrative functions i.e. add friends, match listing, ..etc) Should I add those to this PR as well?

Reply:
Yep please :) 

I'd like to make some (very) small stylistic code changes so add everything you have to this PR and I'll go through it as a whole. Thanks so much!

Reply:
By the way, can you jump on Gitter (https://gitter.im/heroiclabs/nakama) and DM me please :)